src/ngraph/node.cpp

```diff
@@ -333,6 +333,18 @@ void Node::set_output_type(size_t i,
     descriptor.set_partial_shape(partial_shape);
 }
 
+void Node::set_output_max_partial_shape(size_t i, const PartialShape& partial_shape)
```

Comment:
why not use @diyessi 's PR , where you have set_xx_max_xx APIs on inputs/outputs, instead of node.

Reply:
this is to set the output shape to max shape and hide the descriptor mess behind this api in node. We might not need the getter for it probably.

Reply:
These are okay; sometimes output is convenient, sometimes node is convenient.